tempodb/encoding/vparquet3/block_autocomplete.go

```diff
@@ -358,6 +358,8 @@ func createDistinctAttributeIterator(
 			}
 			keyIter = makeIter(keyPath, parquetquery.NewStringInPredicate([]string{cond.Attribute.Name}), selectAs("key", cond.Attribute))
 			valIter = makeIter(boolPath, pred, selectAs("bool", cond.Attribute))
+		default:
+			continue
```

Comment:
Another option is returning an error, but opted for ignoring the condition.

The reasoning is that autocomplete should be a smooth experience to the user. If the filtering query has an error, they'll get that feedback via normal search—ie. when they execute the query.

Reply:
are we missing case statements for things like `kind = server` and `status = ok`?

it seems like we should support auto complete on things like:
```
{ name = "foo" && kind = server && status = error && span.foo = ...
```

Reply:
thx for the clarification.

agree with your initial thought we should just ignore the bad condition over returning an error.